### PR TITLE
Show firmware loaded in firmware flasher tab

### PR DESCRIPTION
**Pull request by haslinghuis**
image

### CODE REVIEW

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:3245]: Now that you are changing it, better to use the ”object" way with {{firmwareSelected}} and {{size}} for example. I think is better for understanding the code and helps translators.

**Review by McGiverGim — CHANGES_REQUESTED** [review]
Good idea!! Only one suggestion...

**Comment by haslinghuis** [comment]
One problem to solve: throwing away the config when loading online target without choosing a new one.

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by daleckystepan** [comment]
Tested on MacOS, works fine.

**Review by ctzsnooze — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**
